pkg/store/bucket.go

```diff
@@ -3325,8 +3334,7 @@ func (r *bucketIndexReader) LoadSeriesForTime(ref storage.SeriesRef, lset *[]sym
 		return false, errors.Errorf("series %d not found", ref)
 	}
 
-	r.stats.seriesTouched++
-	r.stats.SeriesTouchedSizeSum += units.Base2Bytes(len(b))
+	r.stats.add(SeriesTouched, 1, len(b))
```

Comment:
Should the stats refactoring be done in another PR?

Reply:
hmm I thought it was small enough to be part of this PR